src/runtime-rs/crates/hypervisor/src/device/driver/virtio_fs.rs

```diff
@@ -42,14 +49,12 @@ pub struct ShareFsMountConfig {
     pub prefetch_list_path: Option<String>,
 }
 
-#[derive(Debug, Clone)]
-pub struct ShareFsMountDevice {
-    pub config: ShareFsMountConfig,
-}
-
 /// ShareFsDeviceConfig: share fs device config
-#[derive(Debug, Clone)]
-pub struct ShareFsDeviceConfig {
```

Comment:
Same remark as with ShareFsMountDevice, and I'm now realizing that all these changes are squashed in the last commit... :disappointed: 

General good practice is to make sure each commit is buildable (it is even mandatory in other projects like QEMU or the Linux kernel). The obvious gain is to be able to use `git bisect`, which is a powerful maintenance tool, but it doesn't seem to be used with Kata. Another gain, maybe more relevant here, is that it helps to regroup changes that should really go together and separate them from other unrelated changes. This results in PRs that are easier to review, merged faster and it improves the code quality.

Reply:
Thx Greg, same as `ShareFsMountDevice` comments I added.